roles/openshift_certificate_expiry/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+openshift_certificate_expiry_config_base: "/etc/origin"
```

Comment:
If this mean that we currently can not support check customized certificates which maybe not located in the default path like os.path.join(openshift_base_config_path, "master/master-config.yaml")?
eg: 
I set openshift_certificate_expiry_config_base to "/usr/share/crts", but there is no master/master-config.yaml in that path?